src/main/java/org/assertj/core/error/ShouldContainCharSequence.java

```diff
@@ -113,6 +114,24 @@ public static ErrorMessageFactory shouldContain(CharSequence actual, CharSequenc
     return shouldContain(actual, strings, notFound, StandardComparisonStrategy.instance());
   }
 
+  /**
+   * Creates a new <code>{@link ShouldContainCharSequence}</code>.
+   *
+   * @param actual the actual value in the failed assertion.
+   * @param strings the sequence of values expected to be in {@code actual}.
+   * @param comparisonStrategy the {@link ComparisonStrategy} used to evaluate assertion.
+   * @return the created {@code ErrorMessageFactory}.
+   */
+  public static ErrorMessageFactory shouldContainAnyOf(CharSequence actual, CharSequence[] strings,
+                                                       ComparisonStrategy comparisonStrategy) {
+    return new ShouldContainCharSequence("%n" +
+      "Expecting actual:%n" +
+      "  %s%n" +
+      "to contain any of:%n" +
+      "  %s%n",
+      actual, Arrays.toString(strings), comparisonStrategy);
+  }
```

Comment:
This is probably not necessary, there is already a `ShouldContainAnyOf` error - sorry, I should have spotted that and mentioned it in the issue description